pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - packages/*
+  - website
```

Comment:
we can put it in `packages` 👍

Reply:
it is simpler to keep on root. This makes it so that we can have docker context bootstrap from the root of mono-repo. We can build the full monorepo from root and then only have it copy files that matter in final image that we publish.

Reply:
I just followed a common structure we keep in our projects. This way `packages` only refers to only standalone packages and `website` is a service that gets deployed so it uses packages.

Reply:
Got it, I guess that makes sense if the `packages` don't get deployed and are all used by the main package!